diode-server/go.mod

```diff
@@ -20,6 +20,12 @@ require (
 )
 
 require (
+	github.com/alicebob/gopher-json v0.0.0-20200520072559-a9ecdc9d1d3a // indirect
+	github.com/yuin/gopher-lua v1.1.1 // indirect
+)
+
+require (
+	github.com/alicebob/miniredis/v2 v2.33.0
```

Comment:
This one, as not indirect, should be in the `require ()` block for non-indirects.